primitives/src/xcm/mod.rs

```diff
@@ -53,6 +53,7 @@ use pallet_xc_asset_config::{ExecutionPaymentRate, XcAssetLocation};
 mod tests;
 
 pub const XCM_SIZE_LIMIT: u32 = 2u32.pow(16);
+pub const MAX_ASSETS: u32 = 64;
```

Comment:
Is this const not already set somewhere ? Or it's an arbitrary value

Reply:
It was already set in `XcmConfig::MaxAssetsIntoHolding` as `ConstU64<64>`, I just refactor it to be primitives.

Reply:
It's more of a const than a primitive though 🙂 

Might be worth putting all consts into a separate module later on.